### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by skeletondyh** [xujinfan/NASP-codes#2]
Hi,

Your work is very interesting. I have several questions about reproducibility which I hope you can clarify for me:

1. How do you decide hyper-parameters for the search process e.g. learning rate for the architecture?  For the common parameters of both search process and evaluation process e.g. learning rate for \omega, are their values same or different in two processes? 

2. I note in the appendix that you run 3 times and report the average, do you only repeat evaluation process 3 times and search once or you search multiple times? If you search several times, is there any difference between the architectures found?  

Thanks

**Comment by xujinfan** [comment]
Hi, Yuhui

As for question 1, the hyper-parameters are the same as DARTS. We haven’t fine-tuned these hyper-parameters, better performance may be obtained by tuning these. As for question 2, we repeated evaluation process for 3 times. And if you search for multiple times, the results are still stable.

**Comment by skeletondyh** [comment]
It's clear. Thanks
